docs-ref-services/servicebus.md

```diff
@@ -197,7 +347,7 @@ will be made by Python before sending to the RestAPI.
 ```python
 broker_properties = '{"ForcePersistence": false, "Label": "My label"}'
 sent_msg = Message(b'receive message',
-                    broker_properties = broker_properties
+                   broker_properties = broker_properties
```

Comment:
Add a **Next steps** section, provide links to logical next steps (perhaps to the main Service Bus docs, and to the SDK source code).